Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8
# [Core/QuestScripts] Implementation of QuestScripts

Note: OnQuestObjectiveChange will not work for the moment.
How to test this:
-Create a player alliance.
-Raise your level to 5.
-Accept the quest 26391.
-Rewarded or quit the quest and you will see how the 80209 aura is removed.
